src/app/app.component.ts

```diff
@@ -11,7 +12,7 @@ import * as _theme from './shared/chart-and-table/themes/simple';
 export class AppComponent implements OnInit {
 
    public chartTheme = _theme.DEFAULT_THEME;
-   public chartType = 'areaspline';//'spline';//' areaspline';
+   public chartType = GRAPH_TYPE.AREA_SPLINE;
```

Comment:
public chartType: GRAPH_TYPE = GRAPH_TYPE.AREA_SPLINE;

usually I'm using CamelCase for that but feel free to use upper cases